src/SFA.DAS.FindApprenticeshipTraining.Api/Models/GetProviderFeedbackResponse.cs

```diff
@@ -53,25 +53,19 @@ public class GetProviderFeedbackAttributeItem
 
     public class GetProviderFeedbackAttributes
     {
-        public List<string> Strengths { get; private set; }
-        public List<string> Weaknesses { get; private set; }
+        public List<FeedbackAttributeDetail> FeedbackAttributeDetail { get; private set; }
 
         public GetProviderFeedbackAttributes Build(List<GetProviderFeedbackAttributeItem> feedbackAttributeItems)
         {
             return new GetProviderFeedbackAttributes
```

Comment:
Did we want to simplify this object so it's not nested anymore? as rather than it being two properties, it's just one so the nesting seems overkill? Could just be a list of Feedback Attribute Details? There may even be an object that is suitable already mentioned previously as I think that's what it converts from in the first place if you have a look?

Reply:
Updated